TAScheduler/domain/CommandHandler.py

```diff
@@ -209,28 +209,30 @@ def _CreateLabHandler(self, cmd: [str]):
         return 'Lab created'
 
     def _AssignCourseTAHandler(self, cmd: [str]):
-        if self.currentUser is None or not self.currentUser.RoleIn(Role.Administrator, Role.Supervisor):
+        if self.currentUser is None or not self.currentUser.RoleIn(Role.Instructor, Role.Supervisor):
             return 'Must be logged in as an Administrator or Supervisor'
```

Comment:
This will need to be changed

Reply:
Shoot -- yeah, my tests were passing the lab stuff (instead of course) to command handler, that's why they didn't catch this.  Got it.  The rest of the error messages should be okay, but double checking now.